src/main/config/config_master.h

```diff
@@ -56,9 +56,11 @@ typedef struct master_t {
     int8_t yaw_control_direction;           // change control direction of yaw (inverted, normal)
     uint8_t acc_hardware;                   // Which acc hardware to use on boards with more than one device
     uint8_t acc_for_fast_looptime;          // shorten acc processing time by using 1 out of 9 samples. For combination with fast looptimes.
-    uint16_t gyro_lpf;                       // gyro LPF setting - values are driver specific, in case of invalid number, a reasonable default ~30-40HZ is chosen.
+    uint16_t gyro_lpf;                      // gyro LPF setting - values are driver specific, in case of invalid number, a reasonable default ~30-40HZ is chosen.
     uint8_t gyro_sync_denom;                // Gyro sample divider
-    uint8_t gyro_soft_lpf_hz;               // Biqyad gyro lpf hz
+    uint8_t gyro_soft_lpf_hz;               // Biquad gyro lpf hz
+    uint8_t gyro_soft_notch_hz;             // Biquad gyro notch hz
+    uint8_t gyro_soft_notch_q;              // Biquad gyro notch quality
```

Comment:
I suggest using int16_t as you also want to set notch filter above 256hz! Motor noise can go up to 300hz